specs/latest/2.0/index.html

```diff
@@ -2882,6 +2883,37 @@ <h2 class="no-toc">Table of contents</h2>
         The error condition is added to ensure consistent behavior across all platforms.
     </div>
 
+    <h3>Color values from a fragment shader should match the color buffer format</h3>
+
+    <p>
+        Color values written by a fragment shader may be floating-point, signed integer, or unsigned
+        integer. If the values written by the fragment shader do not match the format(s) of the
+        corresponding color buffer(s), the result is undefined in OpenGL ES 3.0  <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-3.9.2">OpenGL ES 3.0.4 &sect;3.9.2</a>)</span>. In WebGL, generates
+        an <code>INVALID_OPERATION</code> error in the corresponding draw call, including <code>
+        clear</code>, <code>drawArrays</code>, <code>drawElements</code>, <code>drawArraysInstanced
```

Comment:
Is it correct to generate INVALID_OPERATION for the clear calls? The fragment shader doesn't run for those, correct?